elixir/apps/domain/lib/domain/relays.ex

```diff
@@ -244,12 +246,13 @@ defmodule Domain.Relays do
       |> Presence.list()
 
     connected_relays = Map.merge(connected_global_relays, connected_account_relays)
-    connected_relay_ids = Map.keys(connected_relays)
+    connected_relay_ids = Map.keys(connected_relays) -- except_ids
 
     relays =
       Relay.Query.not_deleted()
       |> Relay.Query.by_ids(connected_relay_ids)
       |> Relay.Query.global_or_by_account_id(account_id)
+      |> Relay.Query.by_last_seen_at_greater_than(5, "second", :ago)
```

Comment:
Do we really need this if we filter the currently disconnecting one? Can you explain where the race condition actually happens?

Reply:
@thomaseizinger I added this as a preventive measure from too quickly changing the online state, it's not required to fix the race. Will think for a bit if we should go with it or not.